source/links.txt

```diff
@@ -656,34 +656,14 @@
 
 .. _openedx-studio: http://groups.google.com/forum/#!forum/openedx-studio
 
-.. _taking_timed_exams: https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/completing_assignments/timed_exams.html
-
-.. _oe sfd proctored exams: https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/completing_assignments/proctored_exams.html#oe-sfd-proctored-exams
-
-.. _course discussions index: https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/sfd_discussions/index.html
-
-.. _sfd notes: https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/SFD_notes.html
-
-.. _sfd check progress: https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/SFD_check_progress.html
-
-.. _sfd wiki: https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/SFD_wiki.html
-
 .. _pub creating a course run: https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/set_up_course/pub_create_ann_course/pub_create_edit_course_run/index.html
```

Comment:
Maybe make an issue to remove these links and change them to refs in the course staff guide? or remove them if they're partner-guide-specific and we're not using them.